index.js

```diff
@@ -1,8 +1,21 @@
 /*!
- * Copyright (c) 2012-2024 Digital Bazaar, Inc.
+ * Copyright 2024 Digital Bazaar, Inc.
```

Comment:
Was there a reason to remove the range?
```suggestion
 * Copyright 2012-2024 Digital Bazaar, Inc.
```

Reply:
Another accident (or bot failure). Will fix.